modules/apps/frontend-icons/frontend-icons-web/npmscripts.config.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+module.exports = {
+	build: {
+		main: 'src/main/resources/META-INF/resources/js/index.ts',
```

Comment:
:scream: You are using the webpack based build. Does it work? :sweat_smile:

Reply:
Yeah so I copied this over from another module just to see if it worked, and I didn't have any issues, or at least none I noticed. It isn't necessarily needed for this module, so I can remove if we don't want more of these lingering around.

Reply:
Actually maybe it doesnt work. I re-tried from a clean slate and I don't think it is properly building for me. I removed for now as it isn't important for this PR